execution_engine/src/engine_state/mod.rs

```diff
@@ -2553,6 +2553,7 @@ fn should_charge_for_errors_in_wasm(execution_result: &ExecutionResult) -> bool
                 | ExecError::NoActiveContractVersions(_)
                 | ExecError::InvalidContractVersion(_)
                 | ExecError::NoSuchMethod(_)
+                | ExecError::AbstractMethod(_)
```

Comment:
I don't want to devolve to bike shedding, but I pretty strongly feel that  AbstractMethod is not a good term to use here; because that exact term has a specific general meaning, I think that it strongly suggests that there is some kind of inheritance model at play. However, there is no such model here, and the overall semantics are entirely different as well. So, I think it's confusing language.

While I can't think of a _perfect_ name off hand that precisely captures what's going on here, I came up with `SourceMethod` or `TemplateMethod` or perhaps `IntegrantMethod` (part of a whole), by reasoning from effect.